### DISCUSSION TITLE
Exclude all from document driven, opt in if needed

### DISCUSSION THREAD

**Discussion by jjsessa** [nuxt/content#2231] [Q&A]
I only have a small content portion of my project (/resources and /blog) but i want to use document driven mode for the new prefetch performance upgrades, is there a way to opt everything out of document driven mode so i dont need to add 
```js
definePageMeta({
    documentDriven: false
})
```
or is this really the only way to handle this atm

**Comment by oneminch** [comment]
I believe you can set it once in your `nuxt.config.js`. Is that what you were looking for?
```js
export default defineNuxtConfig({
  modules: ['@nuxt/content'],
  content: {
    documentDriven: false
  }
})
```
